static/css/gni.css

```diff
@@ -291,7 +291,7 @@ body {
 }
```

Comment:
Can you make a screenshot on how the current GNI would look like with the css change? Trying not to make too big a difference for it.

Reply:
Here's the screenshot.
https://drive.google.com/file/d/18AKyoYPRmswHM4cU3jRgu9Wyv4bPmAT6/view?usp=sharing